zerver/lib/generateData.py

```diff
@@ -0,0 +1,273 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import itertools
+import json
```

Comment:
We generally use `ujson` exclusively in the project, since it's a bit faster and is a dependency anyway.